src/components/common/Icons.tsx

```diff
@@ -0,0 +1,236 @@
+            fill="#F97316"
+          />
+        </g>
+      </g>
+    </svg>
+  ),
+
+  ArrowSidebarIcon: ({ className, ...props }: SVGProps) => (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 20 20"
+      aria-hidden="true"
+      focusable="false"
+      data-prefix="far"
+      data-icon="arrow-alt-circle-up"
+      role="img"
+      className={twMerge("h-5 w-5 ", className)}
+      {...props}
+    >
+      <g id="Alt Arrow Down">
+        <path
+          id="Vector"
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M3.69213 7.09327C3.91677 6.83119 4.31133 6.80084 4.57341 7.02548L10 11.6768L15.4266 7.02548C15.6887 6.80084 16.0832 6.83119 16.3079 7.09327C16.5325 7.35535 16.5022 7.74991 16.2401 7.97455L10.4067 12.9745C10.1727 13.1752 9.82731 13.1752 9.59325 12.9745L3.75992 7.97455C3.49784 7.74991 3.46749 7.35535 3.69213 7.09327Z"
+          fill="#F96332"
+        />
+      </g>
+    </svg>
+  ),
+
+  OrderByIcon: ({ className, ...props }: SVGProps) => (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 44 44"
+      aria-hidden="true"
+      focusable="false"
+      data-prefix="far"
+      data-icon="arrow-alt-circle-up"
+      role="img"
+      className={twMerge("h-11 w-11 ", className)}
+      {...props}
+    >
+      <g clipPath="url(#clip0_197_14199)">
+        <path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M25.3333 16.375C25.5024 16.375 25.6643 16.4435 25.782 16.5649L29.1153 20.0024C29.3556 20.2502 29.3495 20.6459 29.1017 20.8862C28.8539 21.1265 28.4583 21.1204 28.218 20.8726L25.9583 18.5423L25.9583 27C25.9583 27.3452 25.6785 27.625 25.3333 27.625C24.9881 27.625 24.7083 27.3452 24.7083 27L24.7083 18.5423L22.4487 20.8726C22.2084 21.1204 21.8127 21.1265 21.5649 20.8862C21.3171 20.6459 21.311 20.2502 21.5513 20.0024L24.8846 16.5649C25.0023 16.4435 25.1642 16.375 25.3333 16.375ZM18.6666 16.375C19.0118 16.375 19.2916 16.6548 19.2916 17L19.2916 25.4577L21.5513 23.1274C21.7916 22.8796 22.1873 22.8735 22.4351 23.1138C22.6829 23.3541 22.689 23.7498 22.4487 23.9976L19.1153 27.4351C18.9976 27.5565 18.8357 27.625 18.6666 27.625C18.4976 27.625 18.3357 27.5565 18.218 27.4351L14.8846 23.9976C14.6443 23.7498 14.6504 23.3541 14.8982 23.1138C15.146 22.8735 15.5417 22.8796 15.782 23.1274L18.0416 25.4577L18.0416 17C18.0416 16.6548 18.3215 16.375 18.6666 16.375Z"
+          fill="#EA580C"
+        />
+      </g>
+      <defs>
+        <clipPath id="clip0_197_14199">
+          <rect width="44" height="44" fill="white" />
+        </clipPath>
+      </defs>
+    </svg>
+  ),
+
+  ChevronDownIcon: ({ className, ...props }: SVGProps) => (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 24 24"
+      aria-hidden="true"
+      focusable="false"
+      data-prefix="far"
+      data-icon="arrow-alt-circle-up"
+      role="img"
+      fill="none"
+      className={twMerge("h-11 w-11 ", className)}
+      {...props}
```

Comment:
`{...props}` should always go last, so you can overwrite the other properties. In this case for example, if you passed a `stroke` to `ChevronDownIcon`, it wouldn't be applied since it would be overwritten.